docs/gen-docs/kyma_alpha_delete.md

```diff
@@ -16,6 +16,7 @@ kyma alpha delete [flags]
 
 ```bash
       --concurrency int              Number of parallel processes (default 4)
+      --keep-crds                    Set --keep-crds=true to keep Kyma CRDs during deletion.
```

Comment:
1) Please align the flag description with the other 3 (no period at the end, described as a noun, not starting with a verb).

```suggestion
      --keep-crds                    Flag specifying whether to keep Kyma CRDs during deletion
```

2) Not sure if you do that for booleans, but all the other flags have type specified next to them.